.ci/tm-config.yaml

```diff
@@ -0,0 +1,67 @@
+
+# configure default values for pull request tests
+test:
+  hostprovider: gke
+
+  # uses the garden setup master to deploy gardener to the host clusters
+  gardensetup:
+    revision: "master"
+
+  # deploys gardener with the pr's head commit version
+  gardener:
+    commit:
+      prHead: true
+
+  # gardener extensions to be deployed
+  # will be merged with the default extensions defined in garden setup.
+  gardener-extensions:
+    value: |
```

Comment:
Instead of using hard coded values here it may be a good idea to use [semver ranges](https://github.com/blang/semver#ranges). [Gardener/Gem](https://github.com/gardener/gem) offers great support for this use case. WDYT?

Reply:
Would be a good idea 👍 
Problem is that gem is working with `git clone` which is not suitable for our testmachinery controller.
But I think we could reuse the solve logic and just work with the GitHub Api

Reply:
Alright! Can you already do it in this PR as soon as the semver range feature is supported?

Reply:
We would need some more adjustments in the bot and I possibly in the gems implementation.
Therefore I would keep it out of this PR and raise another one if the semver ranges are supported?
Issue: https://github.com/gardener/test-infra/issues/188

Reply:
I prefer to avoid any possible manual version maintenance like this. The Gardener Extensions are released quite often.
A Gardener maintainer always has to check for the current default value and pass the desired one as a flag (if defaults are outdated). It's error prone and feels kind of unhandy, imho.
As you said, every test run produces costs and this handling might cause the maintainer to start the test multiple times.

If the tm bot is not of a high prio and we can wait for gardener/test-infra#188, we should definitely do it.

Reply:
You're right we do not want any manual interaction.
Do you want to just use the latest stable version like `1.x` or some special meaning `latest` for all extensions?
I will tackle this issue today so let's wait this one day 👍

Reply:
Nice, thank you :)

> Do you want to just use the latest stable version like 1.x or some special meaning latest for all extensions?

I'm actually fine with both but `latest` would be even more convenient.

Reply:
Adapted the extensions to use yaml and latest.
Latest will fetch the latest available release from the given repository.

Note that is is currently not possible to test dev versions e.g. 1.0.0-dev-<hash>.
We would need a follow up with our cicd team to tag images differently.
But I think for the current state it is ok to test only released extensions